test/PositionManager.ts

```diff
@@ -123,18 +171,167 @@ describe("GammaPoolFactory", function () {
+    describe("Long Gamma Functions", function () {
+        it("#createLoan should return tokenId", async function () {
+            expect(tokenId.toNumber()).to.equal(1);
+        });
+
+        it("#borrowLiquidity should return tokenId", async function () {
+            const BorrowLiquidityParams = {
+                cfmm: cfmm.address,
+                protocol: 1,
+                tokenId: 1,
+                lpTokens: 1,
+                to: owner.address,
+                deadline: ethers.constants.MaxUint256
+            }
+            
+            const res = await (await posMgr.borrowLiquidity(BorrowLiquidityParams)).wait();
+            
+            const { args } = res.events[0]
+            expect(args.pool).to.equal(gammaPool.address);
+            expect(args.tokenId.toNumber()).to.equal(1);
+        });
+
+        it("#repayLiquidity should return tokenId, paid liquidity, paid lp tokens and length of amounts array", async function () {
+            const RepayLiquidityParams = {
+                cfmm: cfmm.address,
+                protocol: 1,
+                tokenId: 1,
+                liquidity: 1,
+                to: owner.address,
+                deadline: ethers.constants.MaxUint256
+            }
+            
+            const res = await (await posMgr.repayLiquidity(RepayLiquidityParams)).wait();
+            
+            const { args } = res.events[0]
+            expect(args.pool).to.equal(gammaPool.address);
+            expect(args.tokenId.toNumber()).to.equal(1);
+            expect(args.liquidityPaid.toNumber()).to.equal(24);
+            expect(args.lpTokensPaid.toNumber()).to.equal(25);
+            expect(args.amountsLen.toNumber()).to.equal(9);
+        });
+
+        it("#increaseCollateral should return tokenId and length of tokens held", async function () {
+            await tokenA.approve(posMgr.address, ethers.constants.MaxUint256);//must approve before sending tokens
+            await tokenB.approve(posMgr.address, ethers.constants.MaxUint256);//must approve before sending tokens
+            
+            const AddRemoveCollateralParams = {
+                cfmm: cfmm.address,
+                protocol: 1,
+                tokenId: 1,
+                amounts: [100,10],
+                to: owner.address,
+                deadline: ethers.constants.MaxUint256
+            }
+            
+            const res = await (await posMgr.increaseCollateral(AddRemoveCollateralParams)).wait();
+            
+            const { args } = res.events[2];
+            expect(args.pool).to.equal(gammaPool.address);
+            expect(args.tokenId.toNumber()).to.equal(1);
+            expect(args.tokensHeldLen.toNumber()).to.equal(6);
+        });
 
+        it("#decreaseCollateral should return tokenId and length of tokens held", async function () {
+            const AddRemoveCollateralParams = {
+                cfmm: cfmm.address,
+                protocol: 1,
+                tokenId: 1,
```

Comment:
use tokenId from beforeEach here, don't just use 1. So it should be "tokenId: tokenId,"